drivers/usb/serial/qcserial.c

```diff
@@ -76,6 +76,8 @@ static const struct usb_device_id id_table[] = {
 	{DEVICE_G1K(0x1bc7, 0x900e)},	/* Telit Gobi QDL device */
 
 	/* Gobi 2000 devices */
+	{USB_DEVICE(0x04da, 0x250e)},   /* Panasonic Gobi 2000 QDL device */
+	{USB_DEVICE(0x04da, 0x250f)},   /* Panasonic Gobi 2000 Modem device */
```

Comment:
You used spaces while all the other comments are separated from the code by tabs :)